src/server/scripts/Northrend/Naxxramas/boss_gluth.cpp

```diff
@@ -35,21 +38,28 @@ enum Creatures
 
 Position const PosSummon[3] =
 {
-    {3267.9f, -3172.1f, 297.42f, 0.94f},
-    {3253.2f, -3132.3f, 297.42f, 0},
-    {3308.3f, -3185.8f, 297.42f, 1.58f},
+    { 3267.9f, -3172.1f, 297.42f, 0.94f },
+    { 3253.2f, -3132.3f, 297.42f, 0 },
+    { 3308.3f, -3185.8f, 297.42f, 1.58f },
 };
 
 enum Events
 {
-    EVENT_WOUND     = 1,
+    EVENT_WOUND = 1,
     EVENT_ENRAGE,
     EVENT_DECIMATE,
     EVENT_BERSERK,
     EVENT_SUMMON,
+    EVENT_SEARCH_ZOMBIE,
+    EVENT_KILL_ZOMBIE,
 };
 
-#define EMOTE_NEARBY    " spots a nearby zombie to devour!"
+#define EMOTE_NEARBY    "Gluth spots a nearby zombie to devour!"
+#define EMOTE_DECIMATE  "Gluth decimates all nearby flesh!"
+#define EMOTE_ENRAGE    "Gluth becomes enraged!"
+
+#define SECONDS *1000
+#define MINUTES *60000
```

Comment:
There are already definitions for this, in general it goes by XXXX * MINUTES * IN_MILLISECONDS

example usage: https://github.com/TrinityCore/TrinityCore/blob/d7c53c30fb45f6da015e126aeb8a4a1901f72e58/src/server/scripts/Outland/TempestKeep/Mechanar/boss_mechano_lord_capacitus.cpp#L85